docs/core/tools/sdk-errors/netsdk1130.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "NETSDK1130: Can't reference a Windows Metadata component directly"
+description: How to resolve the issue of not being able to reference a Windows Metadata component directly from an app that targets .NET 5 or later.
+author: tdykstra
+ms.author: tdykstra
+ms.topic: error-reference
+ms.date: 06/29/2021
+f1_keywords:
+- NETSDK1130
+---
+# NETSDK1130: Can't reference a Windows Metadata component directly
+
+NETSDK1130 indicates that you're trying to reference a Windows Metadata component directly from an app that targets .NET 5 or later. The full error message is similar to the following example:
+
+> *\<Component name>* cannot be referenced. Referencing a Windows Metadata component directly when targeting .NET 5 or higher is not supported.
+
+To resolve this error:
+
+* Remove references to the [Microsoft.Windows.SDK.Contracts package](https://www.nuget.org/packages/Microsoft.Windows.SDK.Contracts).  Instead, specify the version of the Windows APIs that you want to access via the `TargetFramework` property of the project.  For example:
+
+  ```xml
+  <TargetFramework>net5.0-windows10.0.19041</TargetFramework>
```

Comment:
This should be `net5.0-windows10.0.19041.0`.

I'd also recommend linking to this page for more info: https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-enhance

I think the ways to resolve this error are essentially the same as NETSDK1149. Maybe it's better to combine the pages?

Reply:
> Maybe it's better to combine the pages?

That might not be ideal for SEO - The error messages are different enough that we wouldn't be able to cover both of them in the title and H1, so I'm inclined to leave them as separate pages.